frontend/src/modules/Servicemap/utils.ts

```diff
@@ -123,9 +125,12 @@ export const getTooltip = (link: {
 							</div>`;
 };
 
-export const transformLabel = (label: string) => {
-	const MAX_LENGTH = 13;
-	const MAX_SHOW = 10;
+export const transformLabel = (label: string, zoomLevel: number) => {
+	//? 13 is the minimum label length. Scaling factor of 0.9 which is slightly less than 1
```

Comment:
NIT: can remove `?` from each comment

Reply:
Thanks for your feedback! I included the //? for improved comments readability,  with the [Better Comments](https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments) extension. However, I'm open to removing it if it doesn't align with the project's style. Let me know your preference.

Reply:
It should be fine. Approved PR. Merging now  ✅